benchexec/tools/swat.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Tool info module for SWAT, a dynamic symbolic execution tool for Java 17.
+    SWAT is currently being developed by the Institute for IT Security at the University of Luebeck ( https://www.its.uni-luebeck.de/en/institute.html ).
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("run-swat.sh")
+
+#    def version(self, executable):
+#        return '42'
+#        #return self._version_from_tool(executable, arg="-v")
+
+    def name(self):
+        return "SWAT"
+
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable] + options
+        if task.property_file:
+            cmd.append(task.property_file)
+        return cmd + list(task.input_files)
+
+    def determine_result(self, run):
+
```

Comment:
Our code formatter would remove this empty line. Please run [Black](https://pypi.org/project/black/) or remove it manually.

Reply:
Blank line is removed.